marketplace/commercelayer-sku/src/index.css

```diff
@@ -0,0 +1,202 @@
+body {
+    overflow: hidden;
+    font-family: "Avenir Next W01", Helvetica, sans-serif;
+}
+
+.name {
+    min-height: 72px;
+    font-weight: 600;
+}
+
+.sku,
+.selected-sku {
+    min-height: 350px;
+    border: 1px solid #eeeeee;
+    margin: 10px;
+    padding: 20px;
+}
+
+.sku {
+    cursor: pointer;
+}
+
+.sku:hover {
+    border: 1px solid #b4c3ca;
+}
+
+.pagination {
+    margin: 5px;
+}
+
+.cf-form-field {
+  margin-bottom: 1.928571428571429em;
+}
+.cf-form-field > label {
+  display: block;
+  margin-bottom: 0.642857142857143em;
+}
+
+input.cf-form-input {
+  font-size: 16px;
+  color: #2a3039;
+  font-family: "Avenir Next W01", Helvetica, sans-serif;
+  display: block;
+  width: 100%;
+  line-height: 1.5;
+  padding: 0 2px;
+  background-color: transparent;
+  border-width: 0 0 1px 0;
+  border-style: solid;
+  border-color: transparent;
+  border-image-width: 0 0 1px 0;
+  border-image-source: url("https://static.contentful.com/app/svg/dotted-border.svg");
```

Comment:
Don't refer Web App assets. Ideally contribute a meaningful reusable component to Forma, less ideally but still better: take the file and put next to this file

Reply:
Took the file and committed it locally.